README.md

```diff
@@ -33,7 +33,7 @@ For that just enter in liferay-portal fodler and run:
 > brew install python
 
 #### Install pip
-> sudo easy_install pip
```

Comment:
so we don't need to do this anymore?

Reply:
and indeed when `pip --version` you can see it